src/FilterAdder.js

```diff
@@ -2,22 +2,35 @@ import _ from 'lodash/fp'
 import React from 'react'
 import { contexturify } from './utils/hoc'
 import { newNodeFromField } from './utils/search'
+import { withTheme } from './utils/theme'
 
 export let fieldsToOptions = _.map(x => ({ value: x.field, ...x }))
 
 let getGroupFields = node => _.map('field', _.getOr([], 'children', node))
 
-let FilterAdder = ({ tree, node, path, fields, Picker, uniqueFields }) => {
+let FilterAdder = ({
+  tree,
+  node,
+  path,
+  fields,
+  theme: { AdderPicker },
+  uniqueFields,
+  label = 'Add Custom Filter',
```

Comment:
With the addition of an `AdderPicker` theme key, and with ModalPicker as the default `AdderPicker`, the only thing left for [ModalFilterAdder](https://github.com/smartprocure/contexture-react/blob/1d1e96a72611c175550f0093b20f9c56a5876b1a/src/ModalFilterAdder.js) to do was to pass a `label` prop into its AdderPicker component. (A regular picker component like NestedPicker has no need for a label, but ModalPicker uses it as the text for the button to open the modal.)

So, let's kill it completely*, and just let the base FilterAdder handle it instead 🙂

<sub>* ModalFilterAdder is still being used in several places that have yet to be refactored, and replacing the internals with this version breaks stuff, so best to hold off on actually killing it for now</sub>

Reply:
+1 on this. Man your comments are getting so professional. Love the footnote.

Reply:
Why not `FilterAdder.Picker`? Isn't this the whole point of having nesting in the themes?